### ISSUE TITLE
(cli) lockfile.rs should have unit tests

### ISSUE THREAD

**Issue by kitsonk** [denoland/deno#7653]
There are not unit tests in `lockfile.rs` 😢  We should have some.

**Comment by summerlinryan** [comment]
I'd love to work on this if possible. This would be my first open source contribution and to Deno. I've read the CONTRIBUTING.md. Let me know if it's alright if I'm the one to work on this 😃.

**Comment by kitsonk** [comment]
Please do.

**Comment by JavascriptMick** [comment]
hi @kitsonk,  rust and opensource beginner here.  I have made a very modest attempt at this here (https://github.com/JavascriptMick/deno/commit/9da7deb4c3a5867057e86bd856605d5e903886e7) .. (branch https://github.com/JavascriptMick/deno/tree/fix-7653).  Any feedback etc would be appreciated before I continue with additional tests.

I was particularly unsure about the use of Tempdir over adding a permanent test file in testdata, this seems to work fine but not sure if there are any gotchas here.

Also error handling of the result is a bit of a mystery.  I have used a match/panic approach in the second test, I think this is ok but not sure.
